pkg/platform/kube/getter.go

```diff
@@ -45,25 +45,27 @@ func (g *getter) get(consumer *consumer, getOptions *platform.GetOptions) ([]pla
 	functions := []platform.Function{}
 	functioncrInstances := []functioncr.Function{}
 
-	// if identifier specified, we need to get a single function
-	if getOptions.Name != "" {
+	// check if identifier specified, if so take identifiers and act accordingly
+	if len(getOptions.MatchCriterias) > 0 {
 
-		// get specific function CR
-		function, err := consumer.functioncrClient.Get(getOptions.Namespace, getOptions.Name)
-		if err != nil {
+		// Iterate for every function name in MatchCriterias
+		for _, matchCriteria := range getOptions.MatchCriterias {
+
+			// get specific function CR
+			function, err := consumer.functioncrClient.Get(getOptions.Namespace,
+				matchCriteria.Name)
 
-			// if we didn't find the function, return an empty slice
-			if apierrors.IsNotFound(err) {
+			// check for unsupported methods, if so, return function found so far
+			if apierrors.IsMethodNotSupported(err) {
```

Comment:
- Why change from IsNotFound to IsMethodNotSupported? We want to ignore the error and continue to the next matchCriteria if a container by these labels wasn't found - this will not cause that.

Summarizing: for each `matchCriteria`:
- Get functionCR
- if we got an error and it's a NotFound error, ignore it and move on to the next matchCriteria
- if the error was any other type, return `nil` and the error wrapped with "Failed to get functions"
- otherwise if there was no error, append to functioncrInstances